frontend/src/features/public-form/components/FormFields/FormFields.tsx

```diff
@@ -104,7 +104,11 @@ export const FormFields = ({
   return (
     <FormProvider {...formMethods}>
       <form onSubmit={formMethods.handleSubmit(onSubmit)} noValidate>
-        <Box bg="white" py="2.5rem" px={{ base: '1rem', md: '2.5rem' }}>
+        <Box
+          bg={isEmpty(formFields) ? 'theme-grey' : 'white'}
```

Comment:
ah @timotheeg i got what you mean. We can probably remove this Box if there are no fields at all. 

supernit/perf: should we also use a length check over the `isEmpty` function to determine emptiness?

Reply:
> supernit/perf: should we also use a length check over the isEmpty function to determine emptiness?

Yes, let's do that! :D

Reply:
done

<details>
<summary>no box</summary>

![image](https://user-images.githubusercontent.com/102740235/182011531-eaccf21f-a96e-4d48-8e44-16cb18a3dc59.png)
</details>

Reply:
also, i noticed this ui inconsistency, is this intended? The opengov logo is touching the rest of the links bar at the footer
![image](https://user-images.githubusercontent.com/102740235/182011601-bc67023b-0a8a-4d3f-95fd-7241c8bc29dc.png)

Reply:
thanks, will add an issue for it to be fixed in future PRs!